### PR TITLE
Add firefox/new/platform.ftl

### PR DESCRIPTION
**Pull request by alexgibson**
Ref: https://github.com/mozilla/bedrock/pull/9683

### CODE REVIEW

File: en/firefox/new/platform.ftl [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+new-platform-the-best-firefox-ever = The best { -brand-name-firefox } ever
+new-platform-lightweight = Lightweight
+new-platform-uses-30-less-memory = Uses 30% less memory than { -brand-name-chrome }
+new-platform-powerfully-private = Powerfully private
+new-platform-truly-private-browsing = Truly Private Browsing with Tracking Protection
+new-platform-advanced-install-options = Advanced install options & other platforms
+new-platform-download-in-another = Download in another language
+new-platform-fix-a-problem = Fix a problem
+new-platform-need-help = Need help?
+new-platform-download-mozilla-firefox-title = Download { -brand-name-mozilla } { -brand-name-firefox }, a free Web browser. { -brand-name-firefox } is created by a global non-profit dedicated to putting individuals in control online. Get { -brand-name-firefox } for { -brand-name-windows }, { -brand-name-mac }, { -brand-name-linux }, { -brand-name-android } and { -brand-name-ios } today!
```
Comment (peiying2) [inline en/firefox/new/platform.ftl:22]: - Capital initial in `Web`.
- `non-profit` to `not-for-profit`

File: en/firefox/new/platform.ftl [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+# /firefox/linux/ HTML page title
+new-platform-download-mozilla-linux = Download { -brand-name-mozilla } { -brand-name-firefox } for { -brand-name-linux }
+
+# /firefox/linux/ HTML page description
+new-platform-download-mozilla-desc-linux = Download { -brand-name-mozilla } { -brand-name-firefox } for { -brand-name-linux }, a free Web browser. { -brand-name-firefox } is created by a global non-profit dedicated to putting individuals in control online. Get { -brand-name-firefox } for { -brand-name-linux } today!
```
Comment (peiying2) [inline en/firefox/new/platform.ftl:28]: - Capital initial in `Web`.
- `non-profit` to `not-for-profit`

File: en/firefox/new/platform.ftl [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+new-platform-your-life-your-business = Your life, your business. { -brand-name-firefox } blocks third-party tracking cookies on { -brand-name-linux }.
+new-platform-speed-meet-security = Speed, meet security. { -brand-name-firefox } is two times faster with 30% less memory than { -brand-name-chrome }.
+new-platform-open-source = Open source
+new-platform-look-under-the-hood = Look under the hood. Like { -brand-name-linux }, { -brand-name-firefox } features are open source.
+
+# /firefox/mac/ HTML page title
+new-platform-download-mozilla-firefox-mac = Download { -brand-name-mozilla } { -brand-name-firefox } for { -brand-name-mac-short }
+
+# /firefox/mac/ HTML page description
+new-platform-download-mozilla-firefox-desc-mac = Download { -brand-name-mozilla } { -brand-name-firefox } for { -brand-name-mac-short }, a free Web browser. { -brand-name-firefox } is created by a global non-profit dedicated to putting individuals in control online. Get { -brand-name-firefox } for { -brand-name-mac-short } today!
```
Comment (peiying2) [inline en/firefox/new/platform.ftl:50]: - Capital initial in `Web`.
- `non-profit` to `not-for-profit`

File: en/firefox/new/platform.ftl [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+new-platform-firefox-doesnt-spy = { -brand-name-firefox } doesn’t spy on searches. We stop third-party tracking cookies and give you full control.
+new-platform-get-speed-and-security = Get speed and security. { -brand-name-firefox } is fast on { -brand-name-mac-short } because we don’t track your moves.
+new-platform-block-trackers = Block trackers
+new-platform-be-the-master-of-your = Be the master of your domain with strict content blocking. Cut off all cookies and trackers.
+
+# /firefox/windows/ HTML page title
+new-platform-download-mozilla-windows = Download { -brand-name-mozilla } { -brand-name-firefox } for { -brand-name-windows }
+
+# /firefox/windows/ HTML page description
+new-platform-download-mozilla-desc-windows = Download { -brand-name-mozilla } { -brand-name-firefox } for { -brand-name-windows }, a free Web browser. { -brand-name-firefox } is created by a global non-profit dedicated to putting individuals in control online. Get { -brand-name-firefox } for { -brand-name-windows } today!
```
Comment (peiying2) [inline en/firefox/new/platform.ftl:71]: - Capital initial in `Web`.
- `non-profit` to `not-for-profit`

**Review by peiying2 — CHANGES_REQUESTED** [review]
@alexgibson Consistent usage of `non-profit` and `Web`. For the web, I remember we changed to capital initial at one point, but didn't we change it back to lower case? 

How do I see the first block of text for Windows? I can see the rest for Linux, Mac and Windows, just not the first part.

**Comment by alexgibson** [comment]
@peiying2 updated thanks

I believe the first block of text are defaults, which some locales fall back to if other strings aren't translated. This is an old page / template, so appoligies if it's not so clear (hard for me to tell as well)

**Review by peiying2 — APPROVED** [review]
LGTM. Thanks for the quick fix!

**Pull request merged**